cluster/gce/util.sh

```diff
@@ -644,10 +644,24 @@ function kube-down {
 
   echo "Bringing down cluster"
 
-  gcloud preview managed-instance-groups --zone "${ZONE}" delete \
+  deleteCmdOutput=$(gcloud preview managed-instance-groups --zone "${ZONE}" delete \
     --project "${PROJECT}" \
     --quiet \
-    "${NODE_INSTANCE_PREFIX}-group" || true
+    "${NODE_INSTANCE_PREFIX}-group" || true)
```

Comment:
Why the "or true"?  I think that you need to keep trying until it returns success?